disparate-flies/WorkforceManagement
# Ay - training details view

Please structure your pull requests this way:

- **Purpose of PR**
To Fulfill ticket #20:
Given user is viewing the list of training programs
When the user clicks on a training program
Then the user should see all details of that training program
And any employees that are currently attending the program

- **Issue number this request impacts** ------
#20 


- **Description of what was changed:**

- Added in StartDate and EndDate to the Training Create Method on the Training Controller
- Added in StartDate and EndDate to the Index of the Training Folder
- Added to Create Method in Training Controller so that only Training Programs set in the future are shown
- Added in SQL statement for Details Method for Training Controller
- Created the Details view in the training folder
- Added in list of employees onto Training Model

- **Thorough list of steps to test**

1. git checkout into master
2. git fetch --all
3. git checkout ay-TrainDetails
4. ls
5. start WorkforceManagement.sln
6. run project
7. in brower - click on Training Programs
8. then click on Details button to see the details of a Training Program (there should be a list of attending employees)



if you don't do this you will be
**FIRED**
